examples/scripts/ct_fan_svmbir_ppp_bm3d_admm_prox.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+"""
+CT Reconstruction (ADMM Plug-and-Play Priors w/ BM3D, SVMBIR+Prox)
+==================================================================
+
+This example demonstrates the use of class
+[admm.ADMM](../_autosummary/scico.optimize.rst#scico.optimize.ADMM) to
+solve a tomographic reconstruction problem using the Plug-and-Play Priors
+framework :cite:`venkatakrishnan-2013-plugandplay2`, using BM3D
+:cite:`dabov-2008-image` as a denoiser and SVMBIR :cite:`svmbir-2020` for
+tomographic projection.
+
```

Comment:
This text should mention the use of the fan beam.

Is there really enough difference with the parallel beam version to justify a different example? (Is it just a change to `geometry="fan"` in a few places?)

Reply:
Fixed it. 
I agree there is not much difference. but it is more than changing the geometry flag to "fan".
There are new options `dist_source_detector` and `magnification`, change in view angle range, more padding in the ground truth, adding `delta_channel` and `delta_pixel` options in the direct svmbir call. (The last two will not be needed once svmbir implements the new defaults).

So I am not sure whether an example is needed or not.

Reply:
So, with the same phantom, make projections and reconstruction with parallel beam and fan beam?

Reply:
Yes, that's what I was thinking. @Michael-T-McCann: Any thoughts?

Reply:
I updated the example to compare fan-beam and parallel-beam reconstructions.

Reply:
On the application side, you usually wouldn't get to choose whether to take fan beam or parallel beam measurements, so a comparison between the two may not be very meaningful. As an alternative, could you take measurements with a slight fan and compare fan and parallel reconstructions? This is something experimentalists do (reconstruct fan geometry with a parallel approximation).

Reply:
Fair point, but I was primarily thinking of contrasting the actual usage of the class in the two different contexts rather than constructing a realistic example.

Reply:
Made the change.